frontend/assets/stylesheets/components/_payment-paypal.scss

```diff
@@ -0,0 +1,351 @@
+
+        @include mq(tablet) {
+            display: table-cell;
+            min-width: gs-span(5);
+            padding: 0;
+            padding-top: $gs-baseline / 3;
+        }
+    }
+
+    .form-section__name-detail {
+        margin-top: 0;
+    }
+
+    .form-panel--no-margin {
+        margin-bottom: 0;
+    }
+
+    .form-panel__edit {
+        @include fs-textSans(1);
+        @include text-link();
+        float: right;
+        cursor: pointer;
+    }
+
+    /* ==========================================================================
+       Forms: Fieldset
+       ========================================================================== */
+
+    .fieldset {
+        border: none;
+        display: block;
+        background-color: inherit;
+    }
+
+    .fieldset__heading {
+        border-top: 1px dotted guss-colour(neutral-3);
+        padding: ($gs-baseline / 4) 0 0;
+        width: 100%;
+    }
+
+    .fieldset__heading--no-border {
+        @include mq(tablet) {
+            padding-top: 0;
+            border: none;            
+        }
+    }
+
+    .fieldset__heading, .fieldset__fields {
+        display: block;
+        margin: 0 0 ($gs-baseline * 2);
+
+        @include mq(tablet) {
+            display: block;
+        }
+    }
+
+    .fieldset__fields {
+        padding: 0;
+        margin-bottom: $gs-baseline * 2;
+        width: 100%;
+
+        @include mq(tablet) {
+            padding-right: $gs-gutter / 2;
+        }
+    }
+
+    .fieldset__headline {
+        @include fs-bodyHeading(2);
+    }
+
+    .payment-method-headline {
+        display: inline-block;
+    }
+
+    /* ==========================================================================
+       Forms: Benefits
+       ========================================================================== */
+
+    .benefits {
+        border: 0;
+        padding: 0;
+        margin: 0;
+        display: table;
+        border-top: 1px solid guss-colour(news-main-2);
+    }
+
+    .benefits__heading {
+        padding: ($gs-baseline / 3) 0 ($gs-baseline * 2);
+
+        @include mq(tablet) {
+            display: table-cell;
+            width: gs-span(4) + $gs-gutter;
+        }
+    }
+
+    .benefits__headline {
+        margin: 0;
+    }
+
+    .benefits__list {
+        vertical-align: top;
+        padding: 0;
+        padding-bottom: $gs-baseline * 2;
+
+        @include mq(tablet) {
+            display: table-cell;
+            width: gs-span(5);
+            padding-top: $gs-baseline / 3;
+        }
+    }
+
+    .benefits__list ul {
+        margin-bottom: 0;
+    }
+
+    .benefits__list li {
+        fill: guss-colour(news-main-2);
+        list-style-image: url(data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB3aWR0aD0iMTIiIGhlaWdodD0iMTIiIHZpZXdCb3g9IjAgMCAxMiAxMiI+PHN0eWxlPi5zdDB7ZmlsbDojNTRDMURBO308L3N0eWxlPjxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjYiIGN5PSI2IiByPSI2Ii8+PC9zdmc+);
+    }
```

Comment:
Agree with all of what you have said @Ap0c. It's worth noting that base64 encoding can make the SVGs chunkier than they need to be, although it doesn't look too bad here. It's not as descriptive as having the markup in the URL IMO.

In [the frontend](https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/static/src/stylesheets/module/experiments/_svg.scss), we have used a function that takes your SVG markup and encodes it (although not in Base64). Perhaps worth running some comparisons in terms of filesize using this method?